api/v1alpha1/edgedeployment_webhook_test.go

```diff
@@ -169,6 +169,9 @@ var _ = Describe("EdgeDeployment Webhook", func() {
 					},
 				}
 			}),
+			table.Entry("auth file label invalid", func() {
```

Comment:
pls add another test for the positive case in which the label  is valid (one with only podman/label and one with ppodman/io.containers.autoupdate.authfile that is mounted properly) (iirc, they were in the previous version of the PR)